### ISSUE TITLE
Cannot install on Mac OSX 10.8 (Mountain Lion) - "Google Refine" is damaged and can't be opened.  You should move it to the Trash.

### ISSUE THREAD

**Issue by tfmorris** [OpenRefine/OpenRefine#590]
_Original author: eliasdab...@gmail.com (July 30, 2012 23:53:39)_

I've just downloaded Google Refine, and trying to install on Mac (running Mountain Lion) It's telling me that it's damaged and should be moved to trash (attached screenshot). 

Appreciate help in installing the app.

_Original issue: http://code.google.com/p/google-refine/issues/detail?id=591_

**Comment by akanik** [comment]
@herrernst xattr worked like a charm. Thanks!

**Comment by BurgosNY** [comment]
@acdha tip also worked for me. I'm on Yosemite (10.10.2). Thanks!

**Comment by sbecainfo** [comment]
thanks @acdha, also worked for me on Yosemite!

**Comment by tongzz** [comment]
thanks @acdha, your method works.

For people who are non-techies like me, open your Terminal on your Mac and type in the command  `codesign -f -s - /Applications/Google\ Refine.app/Contents/MacOS/JavaApplicationStub`

Quoting @acdha 
>Replace the existing signature with an ad-hoc self-signature: `codesign -f -s - /Applications/Google\ >Refine.app/Contents/MacOS/JavaApplicationStub`
>Open /Applications, control-click on the Google Refine icon and choose Open
>Now the warning dialog will have an Open option in addition to Cancel
>After the first time, simply opening the app normally works.

**Comment by tfmorris** [comment]
@felixrabe I just double checked the v2.6-beta1 kit on a fresh Mac and didn't have any issue with it.  I got a message saying that my security preferences only allow apps from identified developers, but after going to Security Preferences, I was given the option to open it anyway.  Nothing about the app being "damaged" as it used to do with 2.5.

Has anyone else had problems with that kit?  We're about to generate the final v2.6 kits and I want to make sure that this problem is actually solved.

**Comment by herrernst** [comment]
Hi, I've tried both beta1 and rc1, and both did work on Yosemite without given the 'damaged' warning.

**Comment by tfmorris** [comment]
@herrernst Thanks very much for the feedback.  I wanted to make sure I wasn't imagining things. :-)

If anyone sees this problem with a recent kit (perhaps on a OS X version), please feel free to re-open, but I'm closing for now.

**Comment by janemnichols** [comment]
Hi,
just tried on my mac and was given the error message. I changed the security setting to allow apps downloaded from anywhere. That worked. (And, changed the setting back after opening OpenRefine). 
![screen shot 2015-05-20 at 10 29 38 am](https://cloud.githubusercontent.com/assets/12533833/7733634/38659f2a-fee5-11e4-9c21-a0c799a31a91.png)
![screen shot 2015-05-20 at 10 29 59 am](https://cloud.githubusercontent.com/assets/12533833/7733638/3a6f35ce-fee5-11e4-8400-43fb67e05572.png)

**Comment by tfmorris** [comment]
@janemnichols Your comment is a little confusing because you mention OpenRefine, but screen shot shows Google Refine, implying that you were running the older Google Refine 2.5 kit.  The problem definitely still exists in 2.5.  It's only OpenRefine 2.6 which will have the fix.

**Comment by janemnichols** [comment]
ok, then! color me red in the face and I'll work w/ the updated fix. Thanks so much for re-directing me and your quick response.

**Comment by tomhundt-navera** [comment]
Confirmed @tfmorris 's solution worked for me -- Refine v2.5 (r2407) on vOSX 10.10.3 -- i.e., whereas it had previously complained, upon trying to run it, the app was "damaged" and I ought to trash it, now it opens it.  Preferences > Security & Privacy > (click the lock and type password to unlock it, then) Allow apps downloaded from Anywhere.  Now it ran when I launched it as usual.  It popped up some kind of firewall warning, asking for permission to accept connections (I gave it).  Up came a browser window showing the app.  That's as far as I've gotten :-)

It says here the OpenRefine versions (which seem to be beta) won't have this "Damaged" issue, but since I'm new to this program I went for the stable release off http://openrefine.org/download.html which got me the same v2.5 Google Refine .dmg file as I got from Google themselves.
